README.md

```diff
@@ -24,7 +24,9 @@ To build and run the production target from the command line:
 ```shell
 docker build --rm -q -f Dockerfile \
     --label "ontotrans.oteapi=production" \
+    --target production \
```

Comment:
This is not needed anymore since the order has been changed back.

Reply:
It makes it explicit however? But you're right - it's not needed as it will be the default now. Do you want to remove it or keep it?

Reply:
It can be kept.